pkgs/top-level/all-packages.nix

```diff
@@ -19765,6 +19765,8 @@ with pkgs;
 
   canon-cups-ufr2 = callPackage ../misc/cups/drivers/canon { };
 
+  hll2390dw-cups = callPackage ../misc/cups/drivers/hll2390dw-cups { };
```

Comment:
Directory name mismatch? - the `default.nix` added here seems to live in ` pkgs/misc/drivers/hll2390dw/default.nix`

Reply:
Well, that's embarassing, I was under the impression that nix-build would've failed, but it seems it either used the system's overlay or the system's packages when building? Unsetting NIX_PATH before nix-building showed very well that it never worked *in that repo*, it worked very well in my overlay though. It is now fixed.